clients/da-vinci-client/src/main/java/com/linkedin/davinci/blob/BlobResponse.java

```diff
@@ -0,0 +1,51 @@
+package com.linkedin.davinci.blob;
+
+import java.util.List;
+import java.util.Objects;
+
+
+public class BlobResponse {
```

Comment:
Let's rename BlobResponse to BlobPeersDiscoveryResponse

Reply:
Made the changes